openfe/setup/atom_mapping/lomap_mapper.py

```diff
@@ -14,9 +14,13 @@ class LomapAtomMapper(LigandAtomMapper):
     time: int
     threed: bool
     max3d: float
+    element_change: bool
+    seed: str
+    shift: bool
 
-    def __init__(self, time: int = 20, threed: bool = True,
-                 max3d: float = 1000.0, element_change: bool = True):
+    def __init__(self, *, time: int = 20, threed: bool = True,
+                 max3d: float = 1000.0, element_change: bool = True,
+                 seed: str = '', shift: bool = False):
```

Comment:
this `shift=False` will give different behaviour to before, but might be better suited to our current available methods?